website/emails/newsletter/2026-06-newsletter.html

```
@@ -0,0 +1,932 @@
+													<span style="color: #191a19; font-size: 36px; line-height: 36px;">•</span>
+												</td>
+												<td style="vertical-align: top; padding-left: 5px; overflow-wrap: break-word;">
+													Matthew's dream map
+													<a style="color: #01579b; text-decoration: none; white-space: nowrap;" href="https://15minutemaps.buzzsprout.com/2525486/episodes/19197513-episode-22-mathew-roberts-where-the-money-flows" target="_blank" rel="noopener noreferrer">
+														<span style="color: #7d8082; font-size: 16px; text-decoration: none;">Podcast 15min Maps</span>
+														<img src="https://cdn.mcauto-images-production.sendgrid.net/044784a789c4863a/6e838a29-1ed9-47a7-acd0-7301e55d55fc/10x10.png" alt="↗" style="height: 10px; vertical-align: middle; margin-left: 1px; display: inline;" />
+													</a>
+												</td>
+											</tr>
+											<tr><td colspan="2" style="height: 2px;"></td></tr>
+											<tr>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </tr> ], start tag match failed [ <tr> ] on line 748.

(tag-pair)